xml/System.Numerics/Vector`1.xml

```diff
@@ -1395,10 +1410,12 @@ The type `T` can be any of the following numeric types:
         <Parameter Name="value" Type="System.Numerics.Vector&lt;T&gt;" Index="0" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="value">The vector to reinterpret.</param>
+        <summary>Reinterprets a <see cref="T:System.Numerics.Vector`1" /> as a new <see cref="T:System.Numerics.Vector`1" />.</summary>
```

Comment:
```suggestion
        <summary>Reinterprets the bits of a source vector into a vector of native-sized integers.</summary>
```

Reply:
Ah good catch: `IntPtr` == `nint`. Much better description than the one with just generics.